Week-1/HTML Fundamentals/Poem Webpage/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Poem Webpage</title>
+  </head>
+  <body>
+    <!-- First stanza of the poem -->
+    <p>
```

Comment:
If you want to have visible spaces between the lines, you can use the `<p>` tag for each line instead of `<br /> `tags. 

```
<p>I'm Nobody! Who are you?</p>
<p>Are you - Nobody - Too?</p>
<p>Then there's a pair of us!</p>
<p>Don't tell! They'd banish us - you know!</p>
```

Reply:
Done, thank you for the feedback